rules/windows/defense_evasion_microsoft_defender_tampering.toml

```
@@ -109,7 +111,11 @@ registry where host.os.type == "windows" and event.type == "change" and process.
     "?:\\Windows\\system32\\svchost.exe", 
     "?:\\Windows\\CCM\\CcmExec.exe", 
     "?:\\Windows\\System32\\DeviceEnroller.exe", 
-    "?:\\Program Files (x86)\\Trend Micro\\Security Agent\\tmuninst.exe"
+    "?:\\Program Files (x86)\\Trend Micro\\Security Agent\\tmuninst.exe",
+    "\\Device\\HarddiskVolume?\\Windows\\system32\\svchost.exe", 
+    "\\Device\\HarddiskVolume?\\Windows\\CCM\\CcmExec.exe", 
+    "\\Device\\HarddiskVolume?\\Windows\\System32\\DeviceEnroller.exe", 
+    "\\Device\\HarddiskVolume?\\Program Files (x86)\\Trend Micro\\Security Agent\\tmuninst.exe"
```

HarddkiskVolume number comes in 1 or 2 or even 3 numbers (from telem): 

```suggestion
    "\\Device\\HarddiskVolume*\\Windows\\system32\\svchost.exe", 
    "\\Device\\HarddiskVolume*\\Windows\\CCM\\CcmExec.exe", 
    "\\Device\\HarddiskVolume*\\Windows\\System32\\DeviceEnroller.exe", 
    "\\Device\\HarddiskVolume*\\Program Files (x86)\\Trend Micro\\Security Agent\\tmuninst.exe"
```